components/metrics/BUILD.gn

```diff
@@ -175,25 +175,25 @@ static_library("metrics") {
 }
 
 if (!use_cobalt_customizations) {
-  # The component metrics provider is a separate target because it depends upon
-  # (the large) component_updater code, and is not needed for some entities that
-  # depend on :metrics.
-  static_library("component_metrics") {
-    sources = [
-      "component_metrics_provider.cc",
-      "component_metrics_provider.h",
-    ]
+# The component metrics provider is a separate target because it depends upon
+# (the large) component_updater code, and is not needed for some entities that
+# depend on :metrics.
+static_library("component_metrics") {
+  sources = [
+    "component_metrics_provider.cc",
+    "component_metrics_provider.h",
+  ]
 
-    public_deps = [
-      "//third_party/metrics_proto",
-    ]
+  public_deps = [
+    "//third_party/metrics_proto",
+  ]
 
-    deps = [
-      ":metrics",
-      "//base",
-      "//components/component_updater",
-    ]
-  }
+  deps = [
+    ":metrics",
+    "//base",
+    "//components/component_updater",
+  ]
+}
```

Comment:
Is this formatting correct? Should be indented one, no?

Reply:
All the modifications here are deliberately to avoid modifying lines from upstream.